lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -29,9 +29,9 @@ class TailoredAudience
                            'accounts/%{account_id}/tailored_audiences' # @api private
     RESOURCE             = "/#{TwitterAds::API_VERSION}/" \
                            'accounts/%{account_id}/tailored_audiences/%{id}' # @api private
-    RESOURCE_USERS       = "/#{TwitterAds::API_VERSION}/ \
-                           accounts/%{account_id}/tailored_audiences/ \
-                           %{id}/users" # @api private
+    RESOURCE_USERS       = "/#{TwitterAds::API_VERSION}/" \
+                           'accounts/%{account_id}/tailored_audiences/' \
+                           '%{id}/users' # @api private
```

Comment:
otherwise, URI will be something like this (take all spaces) and fail:
```
"/5/                            accounts/18ce54t1ol3/tailored_audiences/                            36n4f/users"
```